docs/misc/changelog.rst

```diff
@@ -35,6 +35,7 @@ Bug Fixes:
 - Fixed ``seed()``` method for ``SubprocVecEnv``
 - Fixed a bug ``callback.locals`` did not have the correct values (@PartiallyTyped)
 - Fixed a bug in the ``close()`` method of ``SubprocVecEnv``, causing wrappers further down in the wrapper stack to not be closed. (@NeoExtended)
+- Fixed a bug in the ``generate_expert_traj()`` method in ``record_expert.py`` when using a non-image vectorized environment (@jbarsce)
```

Comment:
please add your username at the bottom too (unless it is already there)

Reply:
done